Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,228 @@
+```
+
+3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday.
+  Sally is instance_of object
+Tuesday is instance_of one-dimensional temporal region 
+Wednesday is an instance of one-dimensional temporal region 
+Arm is an instance of fiat object part
+“Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+“Sally participates in having no arms on Wednesday” is an instance of occurrent
+“Sally participates in having at least one arm on Tuesday precedes Sally participates in having no arms on Wednesday”
+
+  (b) Every liver has some cell as part at all times it exists.
+  Liver has_part_at_all_times Cell
+  
+  (c) John was a child, then an adult, then a senior. 
+  John is an instance_of object
+John's childhood is an instance_of occurrent.
+John's adulthood is an instance_of occurrent.
+John's seniorhood is an instance_of occurrent.
+“John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood.”
+
+  (d) Goofus and Gallant are married at each point in a three year span. 
+  Goofus is an instance_of object
+Gallant is an instance_of object
+Marriage is an instance_of occurent
+“Three years span 1” is an instance_of one-dimensional temporal region.
+If zero-dimensional temporal region t1 is part_of the one-dimensional temporal region “three years span 1”, then Goofus participates in marriage at t1 and Galland participates in marriage at t1.
+
+```
+
+4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday.
+  ∃(x)(x=s^(Tx^~Wx)) 
+  s=Sally 
+  T=Has an arm on Tuesday 
+  W=Has an arm on Wednesday
+  (b) Every liver has some cell as part at all times it exists.
+  ∀(x)(Lx->∃(y)(Cy^Pyx))
+  L=Is a liver 
+  C=Is a cell 
+  P=_is a part of_
+  
+  (c) John was a child, then an adult, then a senior.
+∃t1∃t2∃t3 (C (j, t1) ∧ A (J, t2) ∧ S(J, t3) ∧ E (t1, t2) ∧ E (t2, t3))
+J = John
+E xy = being earlier than
+C (x, t) = being a child at t
+A (x, t) = being an adult at t
+S (x, t) = being a senior at t
+
+  (d) Goofus and Gallant have been married for three years; for each day of that span, it is true to assert they are married.
+  
+∀t(D(t) ∧ Y(t)→(M(g1,t) ∧ M(g2,t)))
```

Comment:
Does this imply they married were at each point throughout the 3 year span? (it should)